rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -276,13 +278,21 @@ error_code sys_rsx_context_free(cpu_thread& cpu, u32 context_id)
 
 	const auto render = rsx::get_current_renderer();
 
-	std::scoped_lock lock(render->sys_rsx_mtx);
+	std::unique_lock lock(render->sys_rsx_mtx);
```

Comment:
sys_rsx.todo above should be sys_rsx.warning since it is now implemented.

Reply:
Nah, it's a hacked implementation, I'll leave it TODO. Also this is a rare syscall that you want to be always noticed warning doesn't make sense here.